### ISSUE TITLE
sevenbridges.errors.BadRequest error when querying files by parent

### ISSUE THREAD

**Issue by zymergen-mbarton** [sbg/sevenbridges-python#111]
When executing the following command:

``` python
api.files.query(parent='folder_name', project='division/project')
```

I get the error:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/mbarton/.venv/lib/python3.6/site-packages/sevenbridges/models/file.py", line 149, in query
    limit=limit, fields='_all', **query_params
  File "/Users/mbarton/.venv/lib/python3.6/site-packages/sevenbridges/meta/resource.py", line 130, in _query
    response = api.get(url=url, params=kwargs)
  File "/Users/mbarton/.venv/lib/python3.6/site-packages/sevenbridges/http/client.py", line 239, in get
    append_base=append_base, stream=stream
  File "/Users/mbarton/.venv/lib/python3.6/site-packages/sevenbridges/decorators.py", line 134, in wrapper
    raise e
sevenbridges.errors.BadRequest: Malformed project query parameter. Expecting ?project=owner/project or ?parent=id.
```

It looks like from the documentation and the error message that these
parameters are mutaully exclusive? I assume the correct way to do this in two
separate queries by getting the folder as an SBG file object first, then doing
a second query with this as a parent?

**Comment by christiansbg** [comment]
Yes, you would do something along this line:

```python
folder = api.files.query(project="division/project", names=["folder_name"])[0]
files = api.files.query(parent=folder, limit=100).all()
for f in files:
    print(f.name)
```

Always make sure to include `.all()`, otherwise you will be served only the first 50 entities by default (I increased it to the max value of 100 above).

**Comment by QuicksilverMachine** [comment]
This is fixed in [0.18.3](https://github.com/sbg/sevenbridges-python/releases/tag/0.18.3). There should be a correct warning when trying to use more than one of project, parent and dataset.
